libbindgen/src/ir/function.rs

```diff
@@ -184,10 +185,12 @@ impl FunctionSig {
             }
         };
 
-        if cursor.kind() == CXCursor_CXXMethod {
-            let is_const = cursor.method_is_const();
-            let is_virtual = cursor.method_is_virtual();
-            let is_static = cursor.method_is_static();
+        let is_method = cursor.kind() == CXCursor_CXXMethod;
+
+        if is_method || cursor.kind() == CXCursor_Constructor {
```

Comment:
Yeah, when we support them that check should be there. But we should also check for virtual destructors, etc.